recipes/main/thai_red_curry.md

```diff
@@ -0,0 +1,41 @@
+# Thai red curry
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-30%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-yellow.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**Head chef:** [@matthewdking](https://github.com/matthewdking/)
+
+**Sous chef:** [@Karyum](https://github.com/Karyum/)
+
+**Date:** 29/01/18, Week 13 FACN3
+
+**Number of servings:** 8
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
+
+## Ingredients
+
+* 2 onions, sliced
+* 4 small sweet potatoes, peeled and chopped into about 2cm cubes
+* 1 large aubergine, chopped into about 2cm cubes
+* 2 red peppers, sliced
+* thai red curry paste
+* 3 cans of coconut milk
+* a bunch of coriander, finely chopped
+
+## Preparation
+
+1. Heat some oil in a large frying pan.
+1. Fry the onions on a medium heat until soft.
+1. Get the sweet potatoes on to boil. You don't want them completely cooked, this is just to start them off.
+1. Add the aubergine to the onions, and fry for about 10 minutes.
+1. When the aubergines are almost cooked add the red peppers.
+1. In a separate pan add some oil, and a couple of tablespoons of the paste. Heat it for a few minutes before adding a can or two coconut milk.
+1. When the paste and coconut milk are completely combined, add the mixture to all of the veg in the big pan.
+1. Leave to simmer. Have a taste of the sauce. Now it is up to the chef to add more paste or coconut milk depending on how spicy you like it. :hot_pepper:
+
+Serve with rice and sprinkle with some chopped coriander.
```

Comment:
how do i make the rice ? some people don't know how to make rice.

Reply:
Maybe we should add a rice recipe that I can reference here? #42